server/views/pages/dailySchedule/dailySchedule.njk

```diff
@@ -158,20 +166,28 @@
                         },
                         {
                             text: "Video link"
-                        },
+                        } if status != 'CANCELLED',
                         {
                             text: "Status"
-                        },
+                        } if status != 'CANCELLED',
+                        {
+                            text: "Cancelled at"
+                        } if status == 'CANCELLED',
+                        {
+                            text: "Cancelled by"
+                        } if status == 'CANCELLED',
                         {
                             text: "Action"
-                        }
-                    ],
+                        } if not (status == 'CANCELLED' and not appointmentsRolledOut)
+                    ] | filter,
                     rows: rows
                 }) }}
             {% else %}
-                <p class="govuk-body">
-                    There are no video appointments scheduled for {{ date | formatDate('iiii d MMMM yyyy') }}
-                </p>
+                {% if status != 'CANCELLED' %}
+                    <p class="govuk-body">There are no video appointments scheduled for {{ date | formatDate('iiii d MMMM yyyy') }}</p>
+                {% else %}
+                    <p class="govuk-body">There are no cancelled video appointments on {{ date | formatDate('iiii d MMMM yyyy') }}</p>
+                {% endif %}
```

Comment:
How similar is form for CANCELLED v. ACTIVE appointments? Just wondered whether a separate form, or a lot of  `if  status != CANCELLED` is best? Your choice though.